cmd/operator-sdk/bundle/validate.go

```diff
@@ -135,19 +150,31 @@ func (c *bundleValidateCmd) addToFlagSet(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.imageBuilder, "image-builder", "b", "docker",
 		"Tool to extract bundle image data. Only used when validating a bundle image. "+
 			"One of: [docker, podman]")
+	fs.StringVarP(&c.outputFormat, "output", "o", output.Text,
+		"Result format for results. One of: [text, json-alpha1]")
+
+	// It is hidden because it is an alpha option
+	// The idea is the next versions of Operator Registry will return a List of errors
+	if err := fs.MarkHidden("output"); err != nil {
+		panic(err)
+	}
 }
 
-func (c bundleValidateCmd) run() error {
+func (c bundleValidateCmd) run() (res output.Result) {
+	// Create Result to be outputted
+	res = output.NewResult()
 
 	logger := log.WithFields(log.Fields{
 		"bundle-dir":     c.directory,
 		"container-tool": c.imageBuilder,
 	})
+
 	val := registrybundle.NewImageValidator(c.imageBuilder, logger)
 
 	// Validate bundle format.
 	if err := val.ValidateBundleFormat(c.directory); err != nil {
-		return fmt.Errorf("invalid bundle format: %v", err)
+		addBundleValidationErrorToResult(err, &res)
```

Comment:
The following would be better:
```suggestion
		addBundleValidationErrorToResult(fmt.Errorf("error validating format in %s: %v", c.directory, err), &res)
```